### ISSUE TITLE
jcmd missing from alpine image

### ISSUE THREAD

**Issue by domsj** [zulu-openjdk/zulu-openjdk#140]
jcmd is missing from the alpine image, despite there being a symlink for it in `/usr/bin/` (so I suppose the omission is not intentional).
jcmd is present in the centos and debian images.

```
jan:~$ docker run --rm -it azul/zulu-openjdk-centos:latest jcmd
1 sun.tools.jcmd.JCmd
jan:~$ docker run --rm -it azul/zulu-openjdk-debian:latest jcmd
1 sun.tools.jcmd.JCmd
jan:~$ docker run --rm -it azul/zulu-openjdk-alpine:latest jcmd
docker: Error response from daemon: failed to create shim: OCI runtime create failed: container_linux.go:380: starting container process caused: exec: "jcmd": executable file not found in $PATH: unknown.
jan:~$ docker run --rm -it azul/zulu-openjdk-alpine:latest sh
/ # jcmd 
sh: jcmd: not found
/ # ls -ahl /usr/bin/jcmd 
lrwxrwxrwx    1 root     root          35 Jan 19 17:33 /usr/bin/jcmd -> ../lib/jvm/default-jvm/jre/bin/jcmd
/ # ls -ahl /usr/lib/jvm/default-jvm/jre/bin/
total 364K   
drwxr-xr-x    2 root     root        4.0K Jan 19 17:33 .
drwxr-xr-x    4 root     root        4.0K Jan 19 17:33 ..
-rwxr-xr-x    1 root     root       14.0K Jan 17 16:58 java
-rwxr-xr-x    1 root     root       14.1K Jan 17 16:58 jfr
-rwxr-xr-x    1 root     root       14.1K Jan 17 16:58 jjs
-rwxr-xr-x    1 root     root       14.1K Jan 17 16:58 keytool
-rwxr-xr-x    1 root     root       14.1K Jan 17 16:58 orbd
-rwxr-xr-x    1 root     root       14.1K Jan 17 16:58 pack200
-rwxr-xr-x    1 root     root       14.1K Jan 17 16:58 policytool
-rwxr-xr-x    1 root     root       14.1K Jan 17 16:58 rmid
-rwxr-xr-x    1 root     root       14.1K Jan 17 16:58 rmiregistry
-rwxr-xr-x    1 root     root       14.1K Jan 17 16:58 servertool
-rwxr-xr-x    1 root     root       14.1K Jan 17 16:58 tnameserv
-rwxr-xr-x    1 root     root      176.8K Jan 17 16:58 unpack200
/ # 
```

**Comment by AntonKozlov** [comment]
Thanks for the report. You could use a full path to the jcmd: 

```
$ docker run --rm -it --pull always azul/zulu-openjdk-alpine:latest sh
latest: Pulling from azul/zulu-openjdk-alpine
Digest: sha256:c72e60513d79f433c03736d25581c3327e7314a0190d5ce2f924b03189005a51
Status: Image is up to date for azul/zulu-openjdk-alpine:latest
/ # ls /usr/lib/jvm/default-jvm/bin/jcmd
/usr/lib/jvm/default-jvm/bin/jcmd
```
```
docker run --rm -it --pull always azul/zulu-openjdk-alpine:latest /usr/lib/jvm/default-jvm/bin/jcmd
```

The problem with the symlink indeed exists in the alpine:3.14, which we use as the base image. It's fixed in alpine:3.15. We are considering the update, but will go slowly to avoid breaking users assuming 3.14.

**Comment by ironstas1k** [comment]
Hey @domsj,
We've moved january zulu release on alpine:3.15, so no need to use this workaround anymore.
